ui/docs/manual/tasks/scopes.mdx

```diff
@@ -25,6 +26,9 @@ In fact, several scope sets must be satisfieid to create a task:
    routes, priorities, and `schedulerId`.
  * The task's `scopes` property
 
-*NOTE*: this means that scopes for a task must often be configured in two places.  For example, to use a scope `secrets:get:org-project-test-creds`, a task associated with a github project would need:
+<Warning>
+*Warning*: 
```

Comment:
This can be omitted, as it's indicated by the component.  (same in a few other locations in the PR)

Reply:
@djmitche Actually the issue stated there are more places other than two stated in the example. I found the following notes which need a warning highlighting for change.
Should I revert the changes and only make changes to the two stated in the example?

Reply:
I appreciate that you've found a few places.  I only meant that `*Warning*:` can be omitted since it's redundant to the newly-added `<Warning>`.